packages/ui/src/comment-board/CommentBoard.styled.js

```diff
@@ -15,6 +15,7 @@ const CommentBoardWrapper = styled.aside`
   top: -8.5px;
   height: 93vh;
   padding: 0 0 20px 0;
+  padding-right: 1.5rem;
```

Comment:
Why this use just padding and set what you want.

Reply:
Okay